Documentation/devel/release.md

```diff
@@ -69,6 +69,22 @@ Now we switch to the GitHub web UI to conduct the release:
 	gpg --detach-sign $NAME.tar.gz
 ```
 
+- Attach each stage1 file individually so they can be fetched by the ACI discovery mechanism. The files must be named as follows:
+
+```
+	cp release-build/bin/stage1-coreos.aci stage1-coreos-v1.1.0-linux-amd64.aci
```

Comment:
It shouldn't include the `v`, we don't include it when we generate the stage1 images.